gnosis-safe-deploy/README.md

```diff
@@ -0,0 +1,49 @@
+# Forta Deployment from a Gnosis Safe
+
+## Deploy a Gnosis Safe to Polygon
+
+Set up the `config.json` file, specifying the addresses of the owners, the number of signatures necessary for a proposal to be approved, and the
```

Comment:
```suggestion
Create a `config.json` file, specifying the addresses of the owners, the number of signatures necessary for a proposal to be approved, and the
```
`config.json` doesn't exist in branch. Maybe create an example file.

Reply:
Created a `config.json.example` file and updated the README.md.